### PR TITLE
Ezpz contact page

### PR DESCRIPTION
**Pull request by KevinLaCarrubba**
- Integrated calendly into the contact page.
`npm i` before you say there is an error.

### CODE REVIEW

File: components/views/contact/Contact.styles.tsx [hunk 1]
```diff
@@ -2,4 +2,79 @@ import styled from '@emotion/styled'
...
+                align-items: center;
+                justify-content: center;
+                img {
+                    width: 100%;
+                }
+            }
+        }
+    }
+
+    @media only screen and (max-width: 1350px) {
```
Comment (arfusop) [inline components/views/contact/Contact.styles.tsx:54]: Just a tip for you, if the mobile styling is only applicable to the `.contact-container`, then put the `@media` at the bottom of that class, not outside of the class then call the class again

**Review by arfusop — CHANGES_REQUESTED** [review]
On the next PR, let's clean up the very close to reusable pattern you have with all of the pages having (nearly) the same layout, css to start at least the top half of the page, with the hero, and then a container below. Let's make this 2 container components/styled components as well

File: components/views/contact/Contact.styles.tsx [hunk 2]
```diff
@@ -2,4 +2,73 @@ import styled from '@emotion/styled'
...
```
Comment (arfusop) [inline components/views/contact/Contact.styles.tsx:54]: I really love how you resolved this convo without ever actually fixing the code. Just trying to shush it away or something. Move the media query inside of the class lol

**Review by arfusop — CHANGES_REQUESTED** [review]
CSS Bugs:
- Home page needs to be fixed on the styling
Screenshot 2023-02-09 at 1 21 13 PM
- Mobile nav bar logo should be clickable, and route to the home page. Pointer=cursor needed as well

File: components/views/contact/Contact.styles.tsx (hunk 2 above)
Comment (KevinLaCarrubba) [reply components/views/contact/Contact.styles.tsx]: possibly fixed

**Review by arfusop — APPROVED** [review]

**Pull request merged**